src/blend.rs

```diff
@@ -60,21 +57,40 @@ type BlendFunction = dyn Fn(f32, f32) -> f32;
 
 /// Returns blend function for given BlendMode
 fn map_blend_mode(blend_mode: BlendMode) -> &'static BlendFunction {
+    // Modes are sorted like in Photoshop UI
     match blend_mode {
         BlendMode::Normal => &normal,
-        BlendMode::Multiply => &multiply,
-        BlendMode::Screen => &screen,
-        BlendMode::Overlay => &overlay,
+        // Here we need Dissolve
```

Comment:
Should we instead add `BlendMode::Dissolve => unimplemented!()` etc for all of these?

Reply:
It can be `BlendMode::Dissolve => &dissolve`
and
```rust
fn dissolve(color_b: f32, color_s: f32) -> f32 {
    unimplemented!()
}
```

Reply:
Even betteer